packages/web/src/js/Collapse.ts

```diff
@@ -1,13 +1,13 @@
 import BaseComponent from './BaseComponent';
-import { enableToggleAutoloader } from './utils/ComponentFunctions';
+import { enableToggleAutoloader, debounce } from './utils/ComponentFunctions';
 import selectorEngine from './dom/SelectorEngine';
```

Comment:
```suggestion
import SelectorEngine from './dom/SelectorEngine';
```
Same as we do it in other files.

Reply:
fixed